### ISSUE TITLE
Extension build files include sourceMap URL, but no sourceMaps

### ISSUE THREAD

**Issue by csvn** [rangle/augury#1169]
Augury version (required): 1.12.0
Angular version (required): 4.1.3
Date: 7 June 2017
OS: Windows

-- Please make sure you're using the latest version of Augury before reporting an issue.

Demo test application:
-- Git repository for demo app showing the issue (optional but very helpful for difficult issues).
-- If a code snippet will completely show the issue, please include it.

Description of issue:
The Augury extension ships with transpiled `.js` files in a `build` folder. These files has `//# sourceMappingURL=ng-validate.js.map` in the bottom, but the `.map` files does not exist in the `build` folder. This means it's possible to encounter a message about `Could not download sourcemap`. Below is a screenshot encountered when using "Debugger for Chrome" in VSCode.

-- Include (clipped) screenshot images if possible.
![image](https://user-images.githubusercontent.com/8770194/26890317-75174106-4bb1-11e7-95d6-1a03c3093638.png)

Steps to reproduce:
Seems a bit overkill, but here's how you can encounter the missing source maps in VSCode:

1. Create a fresh `@angular/cli` project and open in it with VSCode
2. Add a `launch.json` file:
  ```json
  {
    "version": "0.2.0",
    "configurations": [
      {
        "type": "chrome",
        "request": "attach",
        "name": "Attach to Chrome",
        "port": 9222,
        "sourceMaps": true,
        "webRoot": "${workspaceRoot}",
        "sourceMapPathOverrides": {
          "webpack:///*": "/*"
        }
      }
    ]
  }
```
3. Install extension "Debugger for Chrome" in VSCode
4. `npm start` the app, and open Chrome (with  remote debugging on port 9222)
5. Launch the debugger in vscode

Additional details:

**Comment by riavalon** [comment]
I've tried and unfortunately, I'm unable to reproduce this issue following the steps you provided. It seems that the source maps are being loaded correctly. Are you still seeing this issue on your projects @csvn?

**Comment by csvn** [comment]
@riavalon Sorry for the late response, I've been on vacation.

I have updated to v1.14.0, but I still have the same issue. I also tried removing the extension and reinstalling it, but to no effect.

I found the install location, and looked at the files in the `build` folder (`C:\Users\<user>\AppData\Local\Google\Chrome\User Data\Default\Extensions\elgalmkoelokbchhkhacckoklkejnhcd\1.14.0_1\build`).´All `.js` files seem to have a sourcemap comment (e.g. `//# sourceMappingURL=ng-validate.js.map`), but I cannot see any `.js.map` files. This seems to be the root cause to the sourcemap (I cannot verify this conclusevily due to the extension becoming corrupted and inactive if any file is changed).

Not sure what goes wrong with the reproduce steps, but unless my installation goes bonkers, there seems to be missing sourcemaps. If `.js.map` files aren´t included, the sourceMaps could probably be turned off in the build script.

Should I try to create a repo to reproduce this?

**Comment by csvn** [comment]
This is still occurring with v1.15.0. I tried removing and installing Augary. When Augary was uninstalled, no errors occurred when I started the debug task.

![image](https://user-images.githubusercontent.com/8770194/36438873-769cb030-166b-11e8-944b-0c7de3809fcb.png)

The build folder of the extension on my disk only contains `.js` files, no `.js.map` files:
![image](https://user-images.githubusercontent.com/8770194/36439044-f526c206-166b-11e8-80f7-fbd8be10ad44.png)
